modules/sdk-coin-avaxc/src/avaxc.ts

```diff
@@ -358,19 +358,22 @@ export class AvaxC extends AbstractEthLikeNewCoins {
 
   /**
    * Queries avax.network for the token balance of an address
-   * @param {string} address - the AVAXC address
+   * @param {string} walletContractAddress - the AVAXC address
+   * @param {string} tokenContractAddress - the Token contract address
    * @returns {Promise<BigNumber>} address balance
    */
-  async queryAddressTokenBalance(address: string, contractAddress: string): Promise<BN> {
+  async queryAddressTokenBalance(tokenContractAddress: string, walletContractAddress: string): Promise<BN> {
```

Comment:
Ideally we should have same order in abstractEthLikeNewCoins.ts and avaxc.ts Because we call this method in the ccr token recovery function

Either we should change the method params in avaxc or in abstractEthLikeNewCoins